src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+@use "../utils/functions" as *;
+@use "../utils/variables" as *;
+
+.cards {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: count($cards-number);
+  padding: 50px 31px;
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $color-bg;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+  margin-bottom: 46px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__heading {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-txt-primary;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-txt-secondary;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-paragraph {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-txt-primary;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+  }
+
+  &__price-heading {
+    margin: 0;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-txt-secondary;
+  }
+
+  &__price-price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-txt-primary;
+  }
+
+  &__link {
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $color-bg;
+    border: 1px solid transparent;
+    padding: 11px 0;
+    background-color: $color-link;
+    border-radius: 5px;
+    transition: color 0.2s linear;
+  }
+
+  &__link:hover {
+    border-radius: 5px;
+    color: $color-link;
+    border: 1px solid $color-link;
+    background-color: $color-bg;
```

Comment:
checklist point 10

you can move &__link:hover to &__link and use &:hover

Reply:
done